tests/BallotParserTest.php

```diff
@@ -60,4 +60,53 @@ public function testEnforceOneDirection() : void
         $this->expectException(\InvalidArgumentException::class);
         $this->instance->parse("A<B>C");
     }
+
```

Comment:
These multi-directional tests feel a little contrived. Is there a better case that could be used instead?

Reply:
Should I even bother supporting both directions?

Reply:
No, I would be inclined to keep it as simple as possible.  If the format is simpler, it will be easier for other people to build tools that use our ballots.

Reply:
Good call. I will excise the double-direction support.